langs/elm/elm.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "application",
+    "source-directories": [
+        "src"
+    ],
+    "elm-version": "0.19.1",
+    "dependencies": {
+        "direct": {
+            "elm/browser": "1.0.2",
+            "elm/bytes": "1.0.8",
+            "elm/core": "1.0.5",
+            "elm/file": "1.0.5",
+            "elm/html": "1.0.0",
```

Comment:
Is `elm/html` and `elm/browser` really needed?

Reply:
The only package that's _really_ needed is `elm/core`. The others are just there because they are official packages but I agree those 3 are most unlikely to be used on code golf. 

I don't suspect `elm/bytes`, `elm/file`, `elm/random` and `elm/svg` to see much use either. I haven't reviewed every single function they offer though so who knows.